### ISSUE TITLE
Update SPARQL query to align with new schema

### ISSUE THREAD

**Issue by surchs** [neurobagel/api#97]
Once https://github.com/neurobagel/encarta/issues/10 is done, the API query template needs to be updated to correspond to the new schema!

**Comment by surchs** [comment]
Applying the bug/blocker label here (bit unfair) to highlight that things will break if we don't fix it

**Comment by alyssadai** [comment]
Closing this as a duplicate of #94, will merge descriptions
